### PR TITLE
feat(organization): create organization

### PR DESCRIPTION
**Pull request by natashatikhonova**
### Description: 
- modal for creating new organization 
- user can now set a title for a new organization
- user who created organization is automatically set as owner of organization
- toast is sent out when organization is created
NB!: to see organization you have created - check emulator

### Images: 
image
image
image

### CODE REVIEW

**Review by lindtvedtsebastian — CHANGES_REQUESTED** [review]

**Review by fredrbus — APPROVED** [review]

File: next-tavla/pages/api/organization.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    if (!user) return response.status(401).json({ error: 'Unauthorized' })
+    try {
+        switch (request.method) {
+            case 'POST':
+                const oID = await createOrganization(
+                    user.uid,
+                    JSON.parse(request.body).name as string,
+                )
+                return response.status(200).json({ oID: oID })
```
Comment (lindtvedtsebastian) [inline next-tavla/pages/api/organization.ts:21]: kan ha det som oid istedenfor oID

File: next-tavla/pages/organizations/[id].tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+function EditOrganizationPage({
+    loggedIn,
+    id,
+}: {
+    loggedIn: boolean
+    id: string
+}) {
+    console.log(id)
```
Comment (lindtvedtsebastian) [inline next-tavla/pages/organizations/[id].tsx:42]: console.log

File: next-tavla/pages/organizations/[id].tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    return (
+        <div className={classes.root}>
+            <AdminHeader loggedIn={loggedIn} />
+            <Contrast>
+                <ToastProvider>
+                    <div></div>
```
Comment (lindtvedtsebastian) [inline next-tavla/pages/organizations/[id].tsx:48]: Kan legge inn IDen her

File: next-tavla/src/Admin/scenarios/Organizations/components/CreateOrganization.tsx/index.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    const [showModal, openModal, closeModal] = useToggle()
+    const [organizationName, setOrganizationName] = useState('')
+    const router = useRouter()
+    const saveOrganization = async () => {
+        const req = await createOrganizationRequest(organizationName)
+        if (req.status == 200) {
+            const organization = await req.json()
+            router.push(`/organizations/${organization.oID}`)
+        }
+    }
```
Comment (lindtvedtsebastian) [inline next-tavla/src/Admin/scenarios/Organizations/components/CreateOrganization.tsx/index.tsx:21]: Kan kjøre en toast hvis man feiler på å lage en ny org

**Review by lindtvedtsebastian — CHANGES_REQUESTED** [review]

File: next-tavla/src/Admin/scenarios/Organizations/components/CreateOrganization.tsx/index.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            const organization = await req.json()
+            router.push(`/organizations/${organization.oid}`)
+        } else {
+            addToast({
+                title: 'Noe gikk galt',
+                content: 'Kunne ikke opprette organisajon',
+                variant: 'info',
+            })
+        }
+    }
```
Comment (lindtvedtsebastian) [inline next-tavla/src/Admin/scenarios/Organizations/components/CreateOrganization.tsx/index.tsx:29]: Foreslår at vi returnerer tidlig hvis det ikke er 200 

```ts
    const saveOrganization = async () => {
        const req = await createOrganizationRequest(organizationName)

        if (req.status !== 200)
            return addToast({
                title: 'Noe gikk galt',
                content: 'Kunne ikke opprette organisajon',
                variant: 'info',
            })

        const organization = await req.json()
        router.push(`/organizations/${organization.oid}`)
    }
```

**Review by lindtvedtsebastian — APPROVED** [review]

**Pull request merged**